x-pack/plugins/rollup/public/test/client_integration/job_create_review.test.js

```diff
@@ -153,9 +162,10 @@ describe.skip('Create Rollup Job, step 6: Review', () => {
         expect(startMock.http.get).not.toHaveBeenCalledWith(jobStartApiPath); // make sure it hasn't been called
 
         actions.clickSave();
-        // Given the following anti-jitter sleep x-pack/plugins/rollup/public/crud_app/store/actions/create_job.js
-        // we add a longer sleep here :(
-        await new Promise((res) => setTimeout(res, 750));
+
+        // There is a 500 timeout before receiving the response.
+        // To be investigated, this is the only app requiring a timeout to avoid a "weird flicker";
```

Comment:
Could you please open an issue for that (if none exist yet), this seems like a good candidate for tech debt work.

Reply:
Good idea. Done (https://github.com/elastic/kibana/issues/121386)